docs/reference/commandline/rm.md

```diff
@@ -29,40 +29,68 @@ Options:
 
 ## Examples
 
-    $ docker rm /redis
-    /redis
+### Remove a container
 
 This will remove the container referenced under the link
 `/redis`.
 
-    $ docker rm --link /webapp/redis
-    /webapp/redis
+```bash
+$ docker rm /redis
+
+/redis
+```
+
+### Remove a container and its links
```

Comment:
Actually, this only removes the "link" (if it's a legacy link on the default bridge network; the same may not apply when using `--link` on a custom network)